mitodl/micromasters
# Added fields, updated serializer for personal data on profile

#### What are the relevant tickets?
Refs #149 

#### What's this PR do?
 - Adds some fields for personal data in profile, removes year_of_birth to be replaced by date_of_birth
 - Renames fields we don't plan on using to have `edx_` prefix. This includes certain job or school information which we'll deal with in a later PR
 - Populates some of these new fields with information from edX. first_name and last_name are created by putting the first word in first_name and all other words in last_name
 - Adds tests for dashboard `SETTINGS`
 - Some fields removed from serializer which we don't plan on using

#### Where should the reviewer start?
profiles/models.py

#### How should this be manually tested?
Run database migrations, go to `/api/v0/profiles/<username>/` and see the updated fields. You should also be able to PATCH most fields.

#### Any background context you want to provide?
This is part of #149, the UI portion will be in a separate PR
